### ISSUE TITLE
2.3.1 npm release has invalid dependency

### ISSUE THREAD

**Issue by dbkr** [matrix-org/seshat#105]
**Describe the bug**
2.3.1 matrix-seshat package from npm doesn't build: its seshat dependency in `native/Cargo.toml` is `seshat = { version = "2.3.0", path = "../../" }`, ie. it's specifying that path, so will work when the checked out from git and built, but not when downloaded on its own from npm (and is also presumably the wrong version).

2.3.0 had the dependency as `seshat = { version = "2.3.0" }` in the npm package (although not in the git tag, which has `seshat = { version = "2.3.0", path = "../../" }`).

**To Reproduce**
1. `mkdir foo; cd foo`
2. `yarn add matrix-seshat`
3. See:
```
% yarn add matrix-seshat                                    
yarn add v1.22.17
warning package.json: No license field
warning No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
error /Users/dave/temp/seshatmeh/node_modules/matrix-seshat: Command failed.
Exit code: 1
Command: neon build --release
Arguments: 
Directory: /Users/dave/temp/seshatmeh/node_modules/matrix-seshat
Output:
neon info forcing rebuild for new build settings
neon info running cargo
error: failed to get `seshat` as a dependency of package `seshat-node v2.3.1 (/Users/dave/temp/seshatmeh/node_modules/matrix-seshat/native)`

Caused by:
  failed to load source for dependency `seshat`

Caused by:
  Unable to update /Users/dave/temp/seshatmeh/node_modules

Caused by:
  failed to read `/Users/dave/temp/seshatmeh/node_modules/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
neon ERR! cargo build failed

Error: cargo build failed
    at Target.<anonymous> (/Users/dave/temp/seshatmeh/node_modules/neon-cli/lib/target.js:98:27)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/dave/temp/seshatmeh/node_modules/neon-cli/lib/target.js:24:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Expected behavior**
Successful build

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser N/A
 - Version N/A

**Smartphone (please complete the following information):**

**Additional context**

**Comment by poljar** [comment]
Right, I forgot to remove that path dep when doing the npm release.

**Comment by poljar** [comment]
This was fixed in the [2.3.2](https://github.com/matrix-org/seshat/releases/tag/2.3.2) release.
